x/ammswap/handler_test.go

```diff
@@ -331,17 +386,17 @@ func buildRandomMsgRemoveLiquidity(addr sdk.AccAddress) types.MsgRemoveLiquidity
 	return msg
 }
 
-func buildRandomMsgTokenToNativeToken(addr sdk.AccAddress) types.MsgTokenToNativeToken {
+func buildRandomMsgTokenToNativeToken(addr sdk.AccAddress) types.MsgTokenToToken {
```

Comment:
it would be  better to change this function name, such as buildRandomMsgTokenToToken